src/driftpy/user_map/userstats_map.py

```diff
@@ -312,7 +312,7 @@ async def update_user_stat(
         existing_drift_user_stat = self.get(authority_str)
 
         if existing_drift_user_stat:
-            existing_drift_user_stat.unsubscribe()
+            await existing_drift_user_stat.unsubscribe()
```

Comment:
I think there might be more places to put await if we change unsubscribe to async

Reply:
for example in here https://github.com/drift-labs/driftpy/pull/244/files#diff-db0d88744e4a99c55b1bafc4cfef8a70f14a83fe2e7fa0ca95496a6e8a487552R290-R294

Reply:
or here perhaps https://github.com/drift-labs/driftpy/pull/244/files#diff-db0d88744e4a99c55b1bafc4cfef8a70f14a83fe2e7fa0ca95496a6e8a487552R272-R274